src/components/Contribute/Knowledge/Native/DocumentInformation/DocumentInformation.tsx

```diff
@@ -333,31 +321,47 @@ const DocumentInformation: React.FC<Props> = ({
       ) : (
         <>
           <UploadFile onFilesChange={handleFilesChange} />
-          <Button variant="primary" onClick={handleDocumentUpload}>
+          <Button variant="primary" onClick={handleDocumentUpload} isDisabled={uploadedFiles.length === 0}>
             Submit Files
           </Button>
         </>
       )}
 
+      {/* Informational Alert */}
       {alertInfo && (
-        <AlertGroup isToast isLiveRegion>
-          <Alert
-            variant={alertInfo.type}
-            title={alertInfo.title}
-            actionClose={<AlertActionCloseButton onClose={onCloseSuccessAlert} />}
-            actionLinks={
-              alertInfo.link && (
-                <>
-                  <AlertActionLink component="a" href={alertInfo.link} rel="noopener noreferrer">
-                    View it here
-                  </AlertActionLink>
-                </>
-              )
-            }
-          >
-            {alertInfo.message}
-          </Alert>
-        </AlertGroup>
+        <Alert variant={alertInfo.type} title={alertInfo.title} actionClose={<AlertActionCloseButton onClose={() => setAlertInfo(undefined)} />}>
```

Comment:
We can switch the toast alert, so that all the alerts are shown in the top right corner. Here is the example https://github.com/instructlab/ui/blob/main/src/components/Contribute/Skill/Native/index.tsx. 

Not required for this PR, but something we can start using for all future alert.